plugins/katello/nightly/user_guide/content_hosts/index.md

```diff
@@ -172,4 +172,22 @@ To set System Purpose attributes for a host, Navigate to the Content Host Detail
 
 ![System Purpose](/plugins/katello/{{ page.version }}/user_guide/content_hosts/system_purpose.png)
 
+## Setting System Purpose attributes via Bulk Actions
 
+Select the Content Hosts that you wish to change the System Purpose attributes on.
+
+Next select the 'Bulk Actions' button in the top right, and choose 'Manage System Purpose'
+
+In the following modal, we can see at the top, how many Content Hosts will be affected by the change. Select the dropdown next to the attributes you want to change, select the desired value and when satisfied with the options, click the 'Assign' button.
+
+![Bulk System Purpose](/plugins/katello/{{ page.version }}/user_guide/content_hosts/bulk_system_purpose.png)
+
+**Some following things to note about how the behaivor works with the dropdowns:**
+
+* **If 'No change' and 'None' are both selected, 'No change' wins**
+
+* **If both 'No change' and other addon items are selected, 'No change' wins**
+
+* **If both 'None (Clear)' and others are selected, None is ignored and the others win**
```

Comment:
I think there's no need to go into the details of how it behaves when the user selects illogical options such as both No change and None.  Rather, I think the most important things to highlight here are 

* If "No change" is selected, the value will remain what it was
* If "None (Clear)" is selected, any existing values will be erased

Reply:
Fixed